docs/integration.md

```diff
@@ -0,0 +1,106 @@
+
+The endpoint accepts a `POST` request with
+
+#### Request Body
+
+| Variable Name | Type                          | Description         |
+| ------------- | ----------------------------- | ------------------- |
+| id            | string                        | event id            |
+| type          | enum: [EventType](#eventtype) |
+| data          | Object                        | any additional data |
+
+#### EventType
+
+| Value                    | Description                                                                 |
+| ------------------------ | --------------------------------------------------------------------------- |
+| IncomingPaymentCompleted | Incoming payment is complete and doesn't accept any incoming funds anymore. |
+| IncomingPaymentExpired   | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
+| OutgoingPaymentCreated   | Outgoing payment was created.                                               |
+| OutgoingPaymentCompleted | Outgoing payment is complete.                                               |
+| OutgoingPaymentFailed    | Outgoing payment failed.                                                    |
+
+The `backend` package requires an environment variable called `WEBHOOK_URL` which MUST specify this endpoint.
+
+### Event Handlers
+
+#### `IncomingPaymentCompleted`
+
+An [Open Payments](./glossary#open-payments) Incoming Payment was completed, either manually or programmatically, i.e. it does not accept any incoming funds anymore. Any funds already received SHOULD be withdrawn.
+
+- Action: Withdraw liquidity
+
+#### `IncomingPaymentExpired`
+
+An [Open Payments](./glossary#open-payments) Incoming Payment has expired, i.e. it does not accept any incoming funds anymore. Any funds already received SHOULD be withdrawn.
+
+- Action: Withdraw liquidity
+
+#### `OutgoingPaymentCreated`
+
+An [Open Payments](./glossary#open-payments) Outgoing Payment has been created. It requires liquidity to be processed.
+
+- Action: Deposit liquidity
+
+#### `OutgoingPaymentCompleted`
+
+An [Open Payments](./glossary#open-payments) Outgoing Payment was completed, i.e. it won't send any further funds. Any excess liquidity should be withdrawn.
+
+- Action: Withdraw liquidity
+
+#### `OutgoingPaymentFailed`
+
+An [Open Payments](./glossary#open-payments) Outgoing Payment completely of partially failed to send funds and won't re-try sending them. All or any excess liquidity should be withdrawn.
+
+- Action: Withdraw liquidity
+
+## Open Payments
+
+The Rafiki `backend` exposes the [Open Payments](./glossary#open-payments) APIs. The are auth-protected using the [Grant Negotiation Authorization Protocol](./glossary#grant-negotiation-authorization-protocol) (GNAP). While Rafiki comes with a reference implementation of a GNAP server--the `auth` package--an [Account Servicing Entity](./glossary#account-servicing-entity) may implement its own GNAP server.
```

Comment:
```suggestion
The Rafiki `backend` exposes the [Open Payments](./glossary#open-payments) APIs. They are auth-protected using the [Grant Negotiation Authorization Protocol](./glossary#grant-negotiation-authorization-protocol) (GNAP). While Rafiki comes with a reference implementation of a GNAP server--the `auth` package--an [Account Servicing Entity](./glossary#account-servicing-entity) may implement its own GNAP server.
```
>may implement its own GNAP server

Do we want to be suggesting that (at least as Open Payments is today)?

Reply:
Is it suggesting it? I just wanted to mention that there is a possibility.